### ISSUE TITLE
Browser support

### ISSUE THREAD

**Issue by jvc26** [RobertWHurst/KeyboardJS#86]
This forms more of a request than an issue, but there isn't really much of a taste of which browsers/versions are supported in the README. Any chance this could be added as it has significant implications for whether I could use a library in my app.

**Comment by spencerthayer** [comment]
+1

**Comment by RobertWHurst** [comment]
Agreed. I would have used Testling CI for this, but it's now defunct and I haven't found an alternative I like yet. Does anyone have any cross browser testing recommendations?

**Comment by spencerthayer** [comment]
@RobertWHurst I have used http://dalekjs.com/ with some success.

**Comment by RobertWHurst** [comment]
@spencerthayer DalekJS is sweet, but I would like something hosted so I can use a proper range of browsers without needing to fire up windows and download a bunch.

**Comment by ahumphreys87** [comment]
browserstack or saucelabs have been good for me. For selenium tests im a fan of nightwatch or pioneer

**Comment by jakecraige** [comment]
Are there any plans to list the expected browser support? It doesn't seem to be listed anywhere and that makes it difficult to evaluate whether or not I should use this lib

**Comment by RobertWHurst** [comment]
@jakecraige If you need to be absolutely sure and you can't test yourself, then I'd use something else for now. KeyboardJS should work without issue on all evergreen browsers, but until I have a proper method of cross platform CI setup I'm not going to create a list as things may change with new releases of the library and/or browsers.
